packages/firefox/firefox.pacscript

```diff
@@ -0,0 +1,24 @@
+name="firefox"
+version="92.0.1"
+maintainer="aranym <aranym1@pm.me>"
+url="https://download-installer.cdn.mozilla.net/pub/${name}/releases/${version}/linux-x86_64/en-US/${name}-${version}.tar.bz2"
+depends="lsb-release libatk1.0-0 libc6 libcairo-gobject2 libcairo2 libdbus-1-3 libdbus-glib-1-2 libfontconfig1 libfreetype6 libgcc-s1 libgdk-pixbuf-2.0-0 libglib2.0-0 libgtk-3-0 libharfbuzz0b libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb-shm0 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrender1"
```

Comment:
`optdepends` missing.

The PKGBUILD has some
```bash
optdepends=('networkmanager: Location detection via available WiFi networks'
            'libnotify: Notification integration'
            'pulseaudio: Audio support'
            'speech-dispatcher: Text-to-Speech'
            'hunspell-en_US: Spell checking, American English'
            'xdg-desktop-portal: Screensharing with Wayland')
```

Keep in mind the [`PKGBUILD`](https://github.com/archlinux/svntogit-packages/blob/packages/firefox/trunk/PKGBUILD) is for a building script. So ignore the building part mostly. Read the `package` part to see if you missed anything.

Reply:
Should be resolved now.

Reply:
Not really, see my latest change request